helpers/dmesg.py

```diff
@@ -33,6 +33,27 @@ def warn_count(self, warn_str):
         match = re.findall(warn_str, self.log)
         return len(match)
 
+class DmesgStatefulFinder(object):
+
+    def __init__(self):
+        self.node = remote.tempesta
+
+    def start(self):
+        try:
+            l,_ = self.node.run_cmd("dmesg|wc -l")
```

Comment:
`dmesg` prints content of the kernels circular buffer. Lines counting is not reliable way to get the desired portion of kernel log. 

As i said in the chat, there is two situations:
1. buffer is not overflown, so the `Start test: ${CURRENT_TEST}` message can be found in the output. The needed fragment starts from that message.
2. buffer is overflown with oopses, the `Start test: ${CURRENT_TEST}` message can't be found. the whole dmesg log output is needed fragment

No need to have `start()` function, just grab the output in the right way.
P.S. I have recently tested Tempesta in current master against TLS client (wrk), a lot of oopses have appeared, try this to test your patch.